constants/index.ts

```diff
@@ -98,3 +100,9 @@ export const NETWORK_TICKERS: KeyValueT<string> = {
 
 export const NETWORK_IDS: KeyValueT<number> = { XEC: 1, BCH: 2 }
 export const QUOTE_IDS: KeyValueT<number> = { USD: 1, CAD: 2 }
+
+export enum BLOCKCHAIN_CLIENTS_OPTIONS {'grpc', 'chronik'}
```

Comment:
It's best to avoid enums, not a really good TS feature
[Here some reasons why](https://fettblog.eu/tidy-typescript-avoid-enums/)

Reply:
nice read, addressed